charmcraft/commands/store/__init__.py

```diff
@@ -827,6 +827,94 @@ def run(self, parsed_args):
                 lib_data.full_name, lib_data.api, lib_data.patch)
 
 
+class FetchLibCommand(BaseCommand):
+    """Fetch one or more charm libraries."""
+    name = 'fetch-lib'
```

Comment:
This is bikeshedding territory, but I find a disconnect between the command name and what it does (on lib by name versus N in tree)

Reply:
It already went through a couple of design meetings with Mark and Gustavo, so probably it's fine to leave the first version like this, yes.